server/index.js

```diff
@@ -55,9 +53,8 @@ app.post('/favo5', function (req, res) {
   INSERT INTO artists (artistName)
   VALUES ('${addingArtistName}');
 `)
-  connection.query(`
-    SELECT id FROM artists WHERE artistName='${addingArtistName}'
-`, function (error, results, fields) {
+  const fetchArtistQuery = `SELECT id FROM artists WHERE artistName='${addingArtistName}'`
```

Comment:
Consider readability refactors like this. Your future self and coworkers will thank you :)